clientupdate/clientupdate.go

```diff
@@ -938,19 +819,6 @@ func (up *Updater) cleanupOldDownloads(glob string) {
 	}
 }
 
-func msiUUIDForVersion(ver string) string {
```

Comment:
This moves to the existing Windows-specific file, so Windows can still use the uuid package.